forestserviceprototype/static_assets/additional.js

```diff
@@ -0,0 +1,51 @@
+$(document).ready(function(){
+  $('.add_another_permit_holder').click(function(e){
+    e.preventDefault();
+    $(".permit_hide").toggle();
+    $(this).text($(this).text() == "Add Another Permit Holder" ? "Remove Second Permit Holder" : "Add Another Permit Holder");
```

Comment:
Most of these words can be lowercased in keeping with 18F's [Content Guide](https://pages.18f.gov/content-guide/capitalization/).